tcks/microprofile-config/src/test/java/io/quarkus/tck/config/Dummy.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.tck.config;
+
+// We need this class so that target/test-classes is added to the class path?
```

Comment:
What breaks if `target/test-classes` isn't on the classpath?

Reply:
Well, it seems that surefire does not create a test classloader and I think we need this because of how framework classes are loaded. But I can try it again since I changed the code to use the `BootstrapClassLoaderFactory`...

Reply:
It fails without the dummy class. Maybe @stuartwdouglas or @aloubyansky would know why?